### PR TITLE
[RPS-199] Indicate latest publications for series

### PR DESCRIPTION
**Pull request by jacsamell**
Added a link to the latest publication on the series search result.
Also added a seperation on the series page between latest and previous
versions.
Cleaned up some unreference test code for series site page as well.

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/series.ftl [hunk 1]
```diff
@@ -19,18 +19,34 @@
...
+                    <h3 class="flush push--bottom"><@fmt.message key="headers.latest-version"/></h3>
+                    <ul class="simple-list simple-list--publications" data-uipath="ps.series.publications-list.latest">
+                        <@publicationItem publication=publications?first/>
+                    </ul>
+                </#if>
+
+                <#if publications?size gt 1>
+                    <h3 class="flush push--bottom"><@fmt.message key="headers.previous-versions"/></h3>
+                    <ul class="simple-list simple-list--publications" data-uipath="ps.series.publications-list.previous">
+                        <#list publications as publication>
```
Comment (ziemsky) [inline repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/series.ftl]: Hmm, I may be wrong but it appears that, apart of being displayed in the dedicated 'Latest Publication' section the latest publication will also be displayed at the top of the 'Previous Publications' section. Have I misread it?

Comment (jacsamell) [reply repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/series.ftl]: 🙈  Good spot

Comment (jacsamell) [reply repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/series.ftl]: Fixed thanks to @wojtek-oledzki 😁

File: repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/series.ftl [hunk 2]
```diff
@@ -19,18 +19,34 @@
...
+                    <h3 class="flush push--bottom"><@fmt.message key="headers.latest-version"/></h3>
+                    <ul class="simple-list simple-list--publications" data-uipath="ps.series.publications-list.latest">
+                        <@publicationItem publication=publications?first/>
+                    </ul>
+                </#if>
+
+                <#if publications?size gt 1>
+                    <h3 class="flush push--bottom"><@fmt.message key="headers.previous-versions"/></h3>
+                    <ul class="simple-list simple-list--publications" data-uipath="ps.series.publications-list.previous">
+                        <#list publications[1..] as publication>
```
Comment (ziemsky) [inline repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/series.ftl]: Cool find! 👍

**Pull request merged**